src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,106 @@
+package duke;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    private static final String FILEPATH = "data/duke.txt";
```

Comment:
Naming convention here should be consistent, either FILEPATH and DIRPATH or FILE_PATH and DIR_Path.